amida-tech/amida-auth-microservice
# hotfix/password-reset-email-link

If this PR fixes a bug, you _must_ add test cases representative of the bug.
#### What's this PR do?
When a client (such as the orange mobile app), submits a password reset request, the auth service sends out an SMPT email with the link to the page where the user will enter their new password and submit. In that email, the link to that page is

{env var AUTH_SERVICE_REFRESH_TOKEN_ENABLED}
/{the password reset token}

#### Related JIRA tickets:
https://jira.amida.com/browse/ORANGE-478

#### How should this be manually tested?
Setup the auth service, `orange-api`, and `orange-web`.

Set AUTH_SERVICE_PASSWORD_RESET_PAGE_URL = protocol://the-domin-where-you-have-orange-web-running.com/reset-password

Use the orange mobile app's "forgot password" functionality. That will send you an email. Click the link in the email, which will take you to your `orange-web` password reset page. Enter the new password and submit. You should now be able to use this new password.

#### Any background context you want to provide?
This is temporary. The permanent/better solution is captured in https://jira.amida.com/browse/ORANGE-996
